src/test/java/Selenide/Assignment.java

```diff
@@ -2,32 +2,39 @@
 
 import com.codeborne.selenide.Configuration;
 import lombok.extern.slf4j.Slf4j;
+import org.openqa.selenium.By;
 import org.testng.annotations.Test;
 
+import static com.codeborne.selenide.Condition.*;
 import static com.codeborne.selenide.Selectors.*;
 import static com.codeborne.selenide.Selenide.*;
-import static com.codeborne.selenide.Condition.*;
 
 @Slf4j
 public class Assignment {
 
     //Selenide documentation: https://selenide.org/documentation.html
 
+    private final By acceptGoogleTermsSelector = byId("L2AGLb");
+
     @Test
     public void testOne() {
         Configuration.headless = false;
         Configuration.holdBrowserOpen = true;
         Configuration.browser = "chrome";
 
         //TODO: go to https://www.google.com
-        open( "https://www.google.com");
+        open("https://www.google.com");
 
         //TODO: get rid of terms of service
-        $(byText("I agree")).click();
+        // bad: works only for one language
+//        $(byText("I agree")).click();
+        // better: works for all languages, even though id looks generated
+        $(acceptGoogleTermsSelector).click();
```

Comment:
valid point^^ the solution is just here as a quick and dirty reference, since we don't expect the candidates to come up with a perfect solution in 25 minutes and while observed ;)

Reply:
jau, dachte mir deshalb man kann die nicht ganz so gute lösung ja auskommentiert drin lassen, dann kann man schnell erkennen wer die bessere lösung nimmt